environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
Ahh, sorry, I missed the rebase request comment. Do I need to do anything else then?